houston/constants.go

```diff
@@ -0,0 +1,9 @@
+package houston
```

Comment:
@andriisoldatenko this didn't exist from what I could tell, so I was trying to find a good location for it.  What do you think about it in the Houston directory?

Reply:
yeah sgtm.